index.bs

```diff
@@ -206,12 +206,48 @@ spec: webidl; urlPrefix: https://heycam.github.io/webidl/#
       <h4 id='mime-type'>MIME types</h4>
 
       <p>
-        In the context of this specification, a MIME type is also called content
-        type. A <dfn>valid media MIME type</dfn> is a string that is a <a>valid
-        MIME type</a> per [[mimesniff]]. If the MIME type does not imply a
-        codec, the string MUST also have one and only one parameter that is
-        named <code>codecs</code> with a value describing a single media codec.
-        Otherwise, it MUST contain no parameters.
+        To check if a string <var>mimeString</var> is a <dfn>valid media MIME
```

Comment:
This should be named something else more specific for this specification, to avoid confusion with the one from the MIME Sniff spec.